yagpdb-cc-website/docs/info/avatar.md

```diff
@@ -3,6 +3,15 @@ sidebar_position: 2
 title: Avatar CC
 ---
 
+This command allows you to view the avatar of a given user defaulting to yourself.
```

Comment:
```suggestion
This command allows you to view the avatar of a given user, defaulting to yourself.
```
Dunno, feel like this eases the flow and prevents misunderstandings. Feel free to ignore.